components/Circle.js

```diff
@@ -0,0 +1,20 @@
+const React = require('react');
+
+class Circle extends React.Component {
+
+  // TODO: create a shouldComponentUpdate() here
+  shouldComponentUpdate(nextProps) {
+    return Boolean(nextProps.color !== this.props.color);
```

Comment:
Not sure why we're casting something to a `Boolean` here, the `!==` will automatically create a boolean value